app/views/search/_trigger_tacos.html.erb

```diff
@@ -0,0 +1,12 @@
+$.ajax({
```

Comment:
General feedback on the approach. None of it is blocking or should be considered reason to change anything without discussion as I'm not looking for changes so much as conversation to help me decide how I feel about some of this work.

- If we keep this as client-side javascript, I think I'd prefer the console logging for non-error states not be on by default for production-like deployments. I'd listen to reasons to keep it on, but it feels like debug data and thus probably doesn't belong in prod to me.
- I know the GraphQL behavior of this is a bit odd, but I think requesting both phrase and source is probably not useful for non-debugging purposes. As we'll just ignore it all, returning just one field seems better for now (I suspect a simplified REST API in addition to the GraphQL endpoint for use by systems that don't care about returned data may be worth considering for us in the future). As we'll hopefully move Hints into Tacos soon we'll have more opportunities to rethink what to request when we get to that.
- We should move all of these `trigger_` events (search, hints, tacos) to a more modern rails approach. That is out of scope for this work, but my gut reaction was "wtf is this mess" and then I realized it was modeled directly on the code I initially wrote for bento which has worked for several years so maybe it isn't as bad as it feels?

Reply:
After a good discussion, we'll open backlog tickets to refactor all the bento async work to Turbo/Stimulus and during that work consider whether it makes sense to move the GraphQL POST into Rails instead of client-side JS... but for now this works well.